synapse/handlers/room_member.py

```diff
@@ -1138,6 +1143,11 @@ async def _make_and_store_3pid_invite(
         if room_name_event:
             room_name = room_name_event.content.get("name", "")
 
+        room_type = ""
+        room_create_event = room_state.get((EventTypes.Create, ""))
+        if room_create_event:
+            room_type = room_create_event.content.get(EventContentFields.ROOM_TYPE, "")
```

Comment:
I left an open question on the MSC about what to do if the field doesn't exist.

Reply:
From the MSC it says to not include this field if it doesn't exist, so I think we want to default this to `None` if the create event doesn't exist or if the event doesn't have the field.

And then update the identity handler to only send the field if it is none-`None`.